src/services/getPreparedTodos.ts

```diff
@@ -0,0 +1,30 @@
+import { Todo } from '../types/Todo';
+
+export function getPreparedTodos(
+  todos: Todo[],
+  { query, status }: { query: string; status: string },
+) {
+  let preparedTodos = [...todos];
+  const normalizeQuery = query.toLowerCase();
+
+  if (query) {
+    preparedTodos = preparedTodos.filter(todo =>
+      todo.title.toLowerCase().includes(normalizeQuery),
+    );
+  }
+
+  if (status) {
```

Comment:
The check `if (status)` assumes that any non-empty string is a valid status. Consider validating the status against expected values ('active', 'completed') to avoid unexpected behavior.